src/main/java/http/request/HttpRequest.java

```diff
@@ -0,0 +1,91 @@
+package http.request;
+
+import http.EnumHttpMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import util.IOUtils;
+
+public class HttpRequest {
+
+  private static final Logger logger = LoggerFactory.getLogger(HttpRequest.class);
+  private BufferedReader br;
+  private FirstLine firstLine;
+  private HttpRequestHeader httpRequestHeader;
+  private HttpRequestBody httpRequestBody;
+
+  public HttpRequest(InputStream in) {
+    setBufferedReader(in);
+    setFirstLine();
+    setHttpRequestHeader();
+    setHttpRequestBody();
+  }
+
+  public void setBufferedReader(InputStream in) {
+    try {
+      br = new BufferedReader(new InputStreamReader(in, "UTF-8"));
```

Comment:
이미 선언되어 있는 상수를 활용할 수 있네요. 재사용 가능한 상수를 활용할 수 있다면 오타도 줄이고, 추후 언젠가 발생할 변경사항으로 인한 상위호환/하위호환을 보장할 수 있는 등 여러 장점이 있습니다
```suggestion
      br = new BufferedReader(new InputStreamReader(in, StandardCharsets.UTF_8));
```

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/55/commits/b573b8f789881629fb10bda7fe5d97b4b7270a21

반영했습니다